scripts/sql/create.sql

```diff
@@ -0,0 +1,41 @@
+-- Delegatees are account that receive delegation of voting power from Delegators
+CREATE TABLE Delegation
+(
+    token_address     CHAR(20),
+    delegator_address CHAR(20),
+    delegatee_address CHAR(20),
+    proof             VARCHAR,
+    delegated_weight  BIGINT,
+    delegated_block   BIGINT,
+    PRIMARY KEY (token_address,
+                 delegator_address,
+                 delegatee_address)
+);
+CREATE INDEX Delegation_index_delegator
+    ON Delegation (
+                   token_address,
+                   delegator_address
+        );
+
+CREATE INDEX Delegation_index_delegatee
+    ON Delegation (
+                   token_address,
+                   delegatee_address
+        );
+
+-- Delegators delegate their voting power to Delegatees
+CREATE TABLE Delegators
+(
+    token_address     CHAR(20),
+    delegator_address CHAR(20),
+    trie_root         CHAR(32),
+    delegated_block   BIGINT,
+    PRIMARY KEY (token_address, delegator_address)
+);
+
+
+CREATE TABLE Token
```

Comment:
@joigno what is the `deployed_instance` being used for?

Reply:
that is the implementation for upgradeable/proxied token

Reply:
why do you need the implementation contract address, isn't the interaction always done via the proxy?

_Doesn't interaction directly with the implementation behind a proxy defeat the purpose the token owner using the proxy?_

Reply:
I think we can use only the token address and not the `deployed_instance`, maybe the deployed instance can be used in the Front-end for the users to check that they are voting on the current implementation of the Proposal.

Reply:
> the deployed instance can be used in the Front-end for the users to check that they are voting on the current implementation of the Proposal.

That sound like you're suggesting its used to identify the proposal? entirely different to the token (Which is the name of the table).
Can we delete this table then?